packages/components/src/components/common/table/tree/tree.css.ts

```diff
@@ -8,3 +8,7 @@ export const expandButton = style({
     }
   }
 });
+
+export const cellIcon = style({
+  fontSize: '16px'
```

Comment:
maybe 15px would be better, as then the rows would stay at a height of 32px. With 16px they grow to 33px.
But not sure if this is really a problem :)
So I think this change is fine for me 👍